src/styles/navstyles.css

```diff
@@ -66,10 +104,15 @@
     background: var(--color-navyblue);
   }
   
-  .navbar-link {
-    text-decoration: none;
+  .navbar-link, .pages {
```

Comment:
When users hover over the navbar pages, they should be able to see it change color when it is hovered over. Because you wrapped the text around this css class instead of keeping it as part of the <Link> text, you need to add the styling for hovering here. In red is where my cursor was when I snapped this screenshot. As you can see, the text stayed white. Previously, it would change to a blue that is part of our color palette.
<img width="436" alt="image" src="https://github.com/user-attachments/assets/e7551b48-997a-4205-bf70-ca37f6087c4b">

Reply:
Second this as well. It's important to have hover interactions to indicate clickability